lib/mandrill-transport.js

```diff
@@ -23,6 +23,27 @@ MandrillTransport.prototype.send = function(mail, callback) {
   var bccAddrs = addressparser(data.bcc) || [];
   var fromAddr = addressparser(data.from)[0] || {};
   var mandrillOptions = data.mandrillOptions || {};
+  var attachments = [];
+
+  // convert attachments passed to nodemailer to the structure used by mandrill
+  if (mail.message && mail.message.childNodes) {
+    mail.message.childNodes.forEach(function(node) {
+      var headersAsMap = {};
+
+      node._headers && node._headers.forEach(function(headerEntry) {
```

Comment:
there could be multiple attachments, so find wont fit.
can use filter for that though, will give it a quick try and if good will push it